flotta-agent.spec

```diff
@@ -1,5 +1,8 @@
 %define _build_id_links none
 
+%global flotta_user flotta
+%global flotta_uid 10009
```

Comment:
why do we want to use specific uid? what happens when it is already used by different user?

Reply:
should we add the user to the blueprint as well?

Reply:
I think we should

Reply:
We need uid > 1000, so `useradd` creates appropriate `/etc/subuid` entries. We could ask the uid to be [reserved](https://docs.fedoraproject.org/en-US/packaging-guidelines/UsersAndGroups/#_soft_static_allocation). But I think we can uses `getent` to obtain the `uid` from `username`. So I will fix.